app/templates/components/forms/admin/content/pages-form.hbs

```diff
@@ -46,7 +45,7 @@
     {{/ui-dropdown}}
   </div>
   <div class="ui buttons">
-    <button class="ui teal button" type="submit">{{t 'Update Page'}}</button>
-    <button class="ui red button">{{t 'Delete Page'}}</button>
+    <button class="ui teal button" type="submit">{{saveButton}}</button>
```

Comment:
@niranjan94 Since we've to use the same page-form for both the update and create so, I was passing the button text along with the action to set text accordingly depending on whether it's create button or updating existing page details.